R/step.R

```diff
@@ -193,7 +197,14 @@ print.dtplyr_step <- function(x, ...) {
     cat_line(cli::style_bold("Call:   "), expr_text(dt_call(x)))
   }
   cat_line()
-  cat_line(format(as_tibble(dt, .name_repair = "minimal"), n = 6)[-1]) # Hack to remove "A tibble" line
+  cat_line(
+    format(
+      as_tibble(dt, .name_repair = "minimal"),
+      n = n,
+      max_extra_cols = max_extra_cols,
+      max_footer_lines = max_footer_lines
+    )[-1]
+  ) # Hack to remove "A tibble" line
```

Comment:
maybe move this comment up one line now that the [-1] it references isn't on the same line